src/commands/deploy.ts

```diff
@@ -21,13 +21,13 @@ export default class Deploy extends BaseCommand<typeof Deploy> {
     `Deploy a new version of ${chalk.underline('an existing documentation')}
 
 ${chalk.dim('$ bump deploy FILE --doc <your_doc_id_or_slug> --token <your_doc_token>')}
-* Let's deploy a new documentation version on Bump... done
+* Let's deploy a new documentation version on Bump.sh... done
```

Comment:
The logging has changed in the latest upgrade PR :) 

```suggestion
* Let's deploy on Bump.sh... done
```